README.md

```diff
@@ -1 +1,160 @@
-# Rotations
+# Rotations.jl
+
+This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
```

Comment:
A package rename? Sounds good to me

On 9 May 2016 at 13:08, Chris Foster <notifications@github.com> wrote:

> In README.md
> <https://github.com/FugroRoames/Rotations.jl/pull/2#discussion_r62446577>:
>
> > @@ -1 +1,160 @@
> > -# Rotations
> > +# Rotations.jl
> > +
> > +This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
>
> various *three dimensional* rotation parameterizations
>
> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/2/files/fb3a7882257b93afed97cfaa9bf56669bbf410dc#r62446577>
>